msbuild/Xamarin.MacDev.Tasks/Tasks/XcodeCompilerToolTask.cs

```diff
@@ -198,14 +198,18 @@ protected int Compile (ITaskItem [] items, string output, ITaskItem manifest)
 			if (IsTranslated ()) {
 				// we force the Intel (translated) msbuild process to launch ibtool as "Apple"
 				tool = "arch";
-				args.Add ("-arch", "arm64e");
+				args.Add ("-arch");
+				args.Add ("arm64e");
 				args.Add ("/usr/bin/xcrun");
 			} else {
 				tool = "/usr/bin/xcrun";
 			}
 			args.Add (ToolName);
-			args.Add ("--errors", "--warnings", "--notices");
-			args.Add ("--output-format", "xml1");
+			args.Add ("--errors");
+			args.Add ("--warnings");
+			args.Add ("--notices");
```

Comment:
You could use:
```suggestion
			args.AddRange ({ "--errors", "--warnings", "--notices" });
```

Reply:
If you do not use {} for a collection, the compiler will use https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.collectionextensions.addrange?view=net-9.0 

```csharp
args.AddRange ("foo", "bar"); 
```

The above should compile